tools/cloud-build/daily-tests/tests/hpc-enterprise-slurm-v6.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+test_name: hpc-enterprise-slurm-v6
+deployment_name: "enter-{{ build }}"
+# Manually adding the slurm_cluster_name for use in node names, which filters
+# non-alphanumeric chars and is capped at 10 chars.
+slurm_cluster_name: "enter{{ build[0:5] }}"
+zone: europe-west1-d
+cli_deployment_vars:
+  region: europe-west1
+  zone: "{{ zone }}"
+  zones: "[europe-west1-b,europe-west1-c,europe-west1-d]"
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/examples/hpc-enterprise-slurm-v6.yaml"
+network: "{{ deployment_name }}-net"
+
+# Note: Pattern matching in gcloud only supports 1 wildcard.
+login_node: "{{ slurm_cluster_name }}-login-*"
+controller_node: "{{ slurm_cluster_name }}-controller"
+post_deploy_tests:
+- test-validation/test-mounts.yml
+- test-validation/test-partitions.yml
+custom_vars:
+  partitions:
+  - n2
+  - c2
+  - c2d
+  # Disable those partitions for now.
+  # Note the current selected region may not support some of these partitions
+  # consult with https://cloud.google.com/compute/docs/regions-zones/
+  #- c3
```

Comment:
I am wondering, should we consider enabling this ? otherwise why should we keep them here ?

Reply:
Some day, those machine types are prone to stockouts and not available in many zones. I would keep it commented out.